buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -35,14 +38,9 @@ if (applyCodeCoverage) {
   apply(from = "${rootDir}/gradle/jacoco.gradle")
 }
 
-val minJavaVersionSupported = project.findProperty("minJavaVersionSupported")?.let(JavaVersion::toVersion)
-  ?: JavaVersion.VERSION_1_8
-val maxJavaVersionForTests = project.findProperty("maxJavaVersionForTests")?.let(JavaVersion::toVersion)
-
 java {
   toolchain {
-    languageVersion.set(JavaLanguageVersion.of(
-      Math.max(minJavaVersionSupported.majorVersion.toInt(), DEFAULT_JAVA_VERSION.majorVersion.toInt())))
+    languageVersion.set(otelJava.minJavaVersionSupported.map { JavaLanguageVersion.of(Math.max(it.majorVersion.toInt(), DEFAULT_JAVA_VERSION.majorVersion.toInt())) })
```

Comment:
Should this be encapsulated in a method in `OtelJavaExtension`?

Reply:
I think this is specifically the semantics of how we define the build toolchain so is appropriate here (can't come up with a name of a method that would be appropriate without directly defining this implementation detail)